src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/mobile/set_media_clock_timer_test.cc

```diff
@@ -358,6 +358,114 @@ TEST_F(SetMediaClockRequestTest, OnEvent_Canceled) {
   command->on_event(event);
 }
 
+TEST_F(SetMediaClockRequestTest, MessageWithForwardSeekIndicator_TRACK) {
+  MessageSharedPtr msg = CreateMsgParams();
+  mobile_apis::SeekIndicatorType::eType seek_indicator_type =
+      mobile_apis::SeekIndicatorType::TRACK;
+
+  (*msg)[am::strings::msg_params][am::strings::forward_seek_indicator] =
+      seek_indicator_type;
+  std::shared_ptr<SetMediaClockRequest> command(
+      CreateCommand<SetMediaClockRequest>(msg));
+
+  ExpectationsSetupHelper(true);
+
+  command->Run();
+
+  EXPECT_EQ(
+      seek_indicator_type,
+      static_cast<mobile_apis::SeekIndicatorType::eType>(
+          (*msg)[am::strings::msg_params][am::strings::forward_seek_indicator]
+              .asInt()));
+  auto msg_params = (*msg)[am::strings::msg_params];
+  EXPECT_TRUE(msg_params.keyExists(am::strings::forward_seek_indicator));
+  EXPECT_FALSE(msg_params.keyExists(am::strings::back_seek_indicator));
+  EXPECT_FALSE(msg_params.keyExists(am::strings::seek_time));
+}
+
+TEST_F(SetMediaClockRequestTest, MessageWithBackSeekIndicator_TRACK) {
+  MessageSharedPtr msg = CreateMsgParams();
+  mobile_apis::SeekIndicatorType::eType seek_indicator_type =
+      mobile_apis::SeekIndicatorType::TRACK;
+
+  (*msg)[am::strings::msg_params][am::strings::back_seek_indicator] =
+      seek_indicator_type;
+  std::shared_ptr<SetMediaClockRequest> command(
+      CreateCommand<SetMediaClockRequest>(msg));
+
+  ExpectationsSetupHelper(true);
+
+  command->Run();
+
+  EXPECT_EQ(
+      seek_indicator_type,
+      static_cast<mobile_apis::SeekIndicatorType::eType>(
+          (*msg)[am::strings::msg_params][am::strings::back_seek_indicator]
+              .asInt()));
+
+  auto msg_params = (*msg)[am::strings::msg_params];
+  EXPECT_TRUE(msg_params.keyExists(am::strings::back_seek_indicator));
+  EXPECT_FALSE(msg_params.keyExists(am::strings::forward_seek_indicator));
+  EXPECT_FALSE(msg_params.keyExists(am::strings::seek_time));
+}
+
+TEST_F(SetMediaClockRequestTest, MessageWithForwardSeekIndicator_TIME) {
+  MessageSharedPtr msg = CreateMsgParams();
+  mobile_apis::SeekIndicatorType::eType seek_indicator_type =
+      mobile_apis::SeekIndicatorType::TIME;
+  std::uint32_t seek_time = 10;
```

Comment:
For magic numbers is better to use const. And please use `u` postfix to avoid implicit conversion